modules/nickserv/group.c

```diff
@@ -1,4 +1,4 @@
-/*
+VENDOR_STRING/*
```

Comment:
Pretty sure that's not right.  Just a reference for when you revisit this pull request later.